float32[] actual_position
---
int32 channels
int32 samples
int32 sample_rate
int16 adc_bit
uint32[] data # C1 C2 C3 C4 C1 C2 C3 C4 ...